cognite_toolkit/_cdf_tk/storageio/_base.py

```diff
@@ -230,3 +230,19 @@ def get_schema(self, selector: T_Selector) -> list[SchemaColumn]:
 
         """
         raise NotImplementedError()
+
+    @abstractmethod
+    def data_to_row(
```

Comment:
Adding this to the `TableStorageIO` to make it clear that all implementations need to support creating a single row as well as a schema.